docs/install.md

```diff
@@ -27,15 +27,16 @@ support. Information about supported processors is provided below.
 ## Recommended Host System
 
 The required host-system configuration for Private Data Objects is to
-separate the Private Data Objects components from the Sawtooth components.
+separate the Private Data Objects components from the ledger components.
 This means if you want to run PDO on a single physical host, either PDO or the
-Sawtooth will have to run in a separate VM or container. In particular, to run
+ledger will have to run in a separate VM or container. In particular, to run
 PDO in SGX HW mode, the PDO component has to run in an SGX-enabled environment.
 Below installation and configuration instructions will make sure that the host
 and the docker components fullfill this requirement.
 
-Sawtooth (and the PDO transaction processors for Sawtooth) should be run on
-Ubuntu 16.04.
+If Sawtooth ledger is used, then both Sawtooth (and the PDO transaction processors for Sawtooth)
+should be run on Ubuntu 16.04. If Microsoft CCF ledger is used, both CCF and the PDO transaction processor
```

Comment:
Unrelated note: Haven't we been running the PDO TP for Sawtooth on Ubuntu 18.04 recently?

Reply:
thank you for the question. PDO documentation clearly says that "Sawtooth (and the PDO transaction processors for Sawtooth) should be run on Ubuntu 16.04."

Reply:
Sure, but this doesn't mean the document is not outdated, and shouldn't be fixed :) Like I said, this is a comment that is unrelated to CCF. But your edits highlighted this particular statement about Sawtooth, which I believe to be out of date.